backend/routes/index.js

```diff
@@ -1579,10 +1579,10 @@ module.exports = function() {
             entry.entries[0].author = users[entry.entries[0].author]
 
             if (entry.state === 'completed') {
-              entry.entries[0].quant = entry.entries[0].quant.split('$').length > 1 ?parseInt(entry.entries[0].quant.split('$')[1]) : entry.entries[0].quant
+              entry.entries[0].quant = entry.entries[0].quant.toString().split('$').length > 1 ?parseInt(entry.entries[0].quant.toString().split('$')[1]) : entry.entries[0].quant
```

Comment:
this should probably be factored into a simple function that we can call - `cleanAmt(amt)` or something - as there's a lot of repetition of a lot of code going on here.